tests/console/activate_products.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Activates SLE products. The list of products has to be
+# provided by test_data in yaml scheduling file.
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use parent "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use scheduler qw(get_test_suite_data);
+use registration qw(add_suseconnect_product);
+
+sub run {
+    my $test_data = get_test_suite_data();
+    my @products  = @{$test_data->{products}};
+    for my $product (@products) {
+        record_info('Activate product', "Activate \"$product\" product");
+        add_suseconnect_product($product);
```

Comment:
Didn't we agree to use other module which uses `ADDONS` vars.json variable?
To be honest, I would rather include support of test_data there instead of creating new module. WDYT?